pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -139,6 +139,13 @@ func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, cond
 	// Update with Ceph Status
 	cephCluster.Status.CephStatus = toCustomResourceStatus(cephCluster.Status, status)
```

Comment:
I believe `toCustomResourceStatus()` will reset the capacity values. To preserve the previous values on failure, we'll need that method to copy them.

Reply:
In fact, we should probably move all these capacity updates inside that method.

Reply:
Done.